drivers/srmetadata.py

```diff
@@ -633,11 +633,11 @@ def getMetadataToWrite(self, sr_info, vdi_info, lower, upper, update_map, \
             raise
 
     # specific functions, to be implement by the child classes
-    def getVdiInfo(self, Dict, generateSector=0):
-        return
+    def getVdiInfo(self, Dict, generateSector=0) -> str:
+        raise NotImplementedError()
```

Comment:
This will cause failures, the correct resolution was in #607 so just drop this and the similar change in `getSRInfoForSectors`